src/styles/styles.tsx

```diff
@@ -0,0 +1,53 @@
+/** @format */
```

Comment:
does it make sense to ignore styles from test cases(jest config)?

Reply:
These styles are used by components in the page, so while the file was a way to seperate a list of styles, they're still part of the objects required to render root.component.tsx